docs/competition/volunteers/refreshments.md

```diff
@@ -1,22 +1,24 @@
 # Refreshments
 
-It's important that volunteers stay hydrated, which can be difficult for those in roles which don't allow them to move. (Don't give refreshments to competitors).
+It's important that volunteers stay nourished and hydrated, which can be difficult for those in roles which don't allow them to move. (Don't give refreshments to competitors).
 
-**In SR2022, the refreshments are in the Volunteer Rest room, which is on level 2 past the toilets and to the left. Also water points are on both level 2 and 3 in the corner furthest from the toilets.**
+**In SR2023, the refreshments are in the Volunteer Rest room, which is on level 2 past the toilets and to the left. Also water points are on both level 2 and 3 in the corner furthest from the toilets.**
 
-It is the responsibility of the refreshments role to offer and provide refreshments to all volunteers, especially those who can't leave their roles.
+It is the responsibility of the refreshments role to offer refreshments to all volunteers, especially those who can't leave their roles.
 
 To do this, you should grab a tray, fill it with a mix of drinks and food, and go to all volunteer positions, offering refreshments.
 Please also fill some cups with water from a nearby water fountain, and brew coffee/tea if available.
 
-People at the following locations are usually unable to leave their posts to fetch refreshments for themselves, so it is important to take refreshments to them regularly:
+People at these locations usually can't leave their posts to fetch refreshments for themselves, so it is important to take refreshments to them regularly:
```

Comment:
Do we want to say something about there possibly being roles which may not be able to acknowledge the refreshments (e.g: livestream) and what to do (maybe just leave them some bottles of water if they don't have any, plus a selection of biscuits)?